tox.ini

```diff
@@ -25,50 +24,44 @@ passenv =
 [testenv:fmt]
 description = Apply coding style standards to code
 deps =
-    black
-    isort
+    black==22.12.0
+    ruff==0.0.206
 commands =
-    isort {[vars]all_path}
+    ruff --fix {[vars]all_path}
     black {[vars]all_path}
 
 [testenv:lint]
 description = Check code against coding style standards
 deps =
-    black
-    flake8-docstrings
-    flake8-builtins
-    pyproject-flake8
-    pep8-naming
-    isort
-    codespell
+    black==22.12.0
+    codespell==2.2.2
+    ruff==0.0.206
 commands =
-    # uncomment the following line if this charm owns a lib
-    # codespell {[vars]lib_path}
-    codespell {toxinidir}/. --skip {toxinidir}/.git --skip {toxinidir}/.tox \
-      --skip {toxinidir}/build --skip {toxinidir}/lib --skip {toxinidir}/venv \
-      --skip {toxinidir}/.mypy_cache --skip {toxinidir}/icon.svg
-    # pflake8 wrapper supports config from pyproject.toml
-    pflake8 {[vars]all_path}
-    isort --check-only --diff {[vars]all_path}
+    codespell {toxinidir}/. \
+    --skip {toxinidir}/.git \
```

Comment:
"Unwrapped" args should be indented so it's clear they go with line 40.

Reply:
@dvdgomez, if you want to address this I think we will have resolved all comments and be clear to merge. Thanks!